packages/venia-ui/lib/components/CommunicationsPage/communicationsPage.css

```diff
@@ -0,0 +1,50 @@
+.root {
+    padding: 20px;
+    font-size: 0.875rem;
+}
+
+.header {
+    padding-bottom: 10px;
+}
+
+.title {
+    margin-bottom: 25px;
+    text-align: center;
+    font-weight: 700;
+}
+
+.message {
+    background-color: rgb(var(--venia-grey));
+    border-radius: 4px;
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    padding: 1rem;
+}
+
+.message:empty {
+    display: none;
+}
+
+.actions {
+    display: grid;
+    justify-items: left;
+    margin-top: 1rem;
+}
+
+.error {
+    composes: message;
+    color: rgb(var(--venia-error));
+}
+
+.form {
+    margin-top: 15px;
+}
+
+.subscribe {
+    margin-top: 25px;
+}
+
+.subscribe_label {
```

Comment:
This is not well documented and we have a lot of cleanup to do, but we have a preference for `camelCased` names, while using underscores for modifiers. ie. `myPage` and `myPage_disabled`.